pennylane/transforms/batch_params.py

```diff
@@ -37,6 +37,37 @@ def _nested_stack(res):
     return tuple(stacked_results)
 
 
+def _split_operations(ops, params, indices, start_index, num_tapes):
```

Comment:
I'm a little confused about the difference between `indices` and `start_index`.

Reply:
`indices` corresponds to which parameters of the tape we want to unbatch, where each entry `i` of `indices` corresponds to accessing the tape parameter at `tape._par_info[i]`.

`start_index` represents the above index for the first parameter of the first operation in `ops`. This is necessary if a tape has a prep, in which case the first few parameters in `params` corresponds to the prep and not the non-prep operations. This is only because we need to call `_split_operations` twice: once for the preps and once for the non-prep operations.

Reply:
Maybe `indices` -> `split_indices` and `start_index` -> `first_op_index`? I can definitely clarify this in the docstring as well.